### PR TITLE
Maybe simpler naming of commands for toggeling the panels? :)

### PR DESCRIPTION
**Pull request by predragnikolic**
I know, I am nitpicking,

But I just can't seem to remember the new commands for toggling the panel. :man_facepalming: 
Can we name them with less words and maybe rearrange the words to be more easy to remeber? :D

### CODE REVIEW

File: Commands/Default.sublime-commands [hunk 1]
```diff
@@ -60,11 +60,11 @@
         "args": {}
     },
     {
-        "caption": "LSP: Toggle Panel: Language Servers",
+        "caption": "LSP: Toggle Log Panel",
```
Comment (predragnikolic) [inline Commands/Default.sublime-commands]: Maybe event "LSP: Toggle Console"

Comment (rwols) [reply Commands/Default.sublime-commands]: I'm okay with any name :)

**Comment by rwols** [comment]
Hmm, it seems the travis reporting mechanism is broken. It is still building pull-requests though.

**Comment by predragnikolic** [comment]
Should I close this?

**Comment by rchl** [comment]
I think new names are better.

**Pull request merged**